testing/go/create_function_test.go

```diff
@@ -871,3 +871,120 @@ $$ LANGUAGE plpgsql;`,
 		},
 	})
 }
+
+func TestCreateFunctionsLanguageSQL(t *testing.T) {
```

Comment:
Let's make a `create_function_sql_test.go` file, and we can rename this current one to `create_function_plpgsql_test.go`, and copy this function to the other file since I can imagine this getting much bigger over time.